pkg/ctl/cmdutils/configfile.go

```diff
@@ -315,3 +317,16 @@ func NewDeleteNodeGroupLoader(rc *ResourceCmd, ng *api.NodeGroup, ngFilter *Node
 
 	return l
 }
+
+// NewUtilsEnableLoggingLoader will load config or use flags for 'eksctl utils enable-logging'
+func NewUtilsEnableLoggingLoader(rc *ResourceCmd) ClusterConfigLoader {
+	l := newCommonClusterConfigLoader(rc)
+
+	l.flagsIncompatibleWithConfigFile.Insert(
+		append(api.SupportedCloudWatchClusterLogTypes(), "all")...,
```

Comment:
Please be sure to update this with new flags. The point of this is to prevent user from using `--config-file` along with flags that map to config fields.